Blogger.Sample/Blogger.Sample.Windows/Blogger.Sample.Windows.csproj

```diff
@@ -131,38 +132,35 @@
     </Page>
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="GalaSoft.MvvmLight.Extras.Win8">
-      <HintPath>..\..\packages\MvvmLightLibs.4.3.31.1\lib\netcore45\GalaSoft.MvvmLight.Extras.Win8.dll</HintPath>
+    <Reference Include="GalaSoft.MvvmLight">
+      <HintPath>..\..\packages\MvvmLightLibs.5.2.0.0\lib\windows81\GalaSoft.MvvmLight.dll</HintPath>
     </Reference>
-    <Reference Include="GalaSoft.MvvmLight.Win8">
-      <HintPath>..\..\packages\MvvmLightLibs.4.3.31.1\lib\netcore45\GalaSoft.MvvmLight.Win8.dll</HintPath>
+    <Reference Include="GalaSoft.MvvmLight.Extras">
+      <HintPath>..\..\packages\MvvmLightLibs.5.2.0.0\lib\windows81\GalaSoft.MvvmLight.Extras.dll</HintPath>
     </Reference>
-    <Reference Include="Google.Apis, Version=1.9.3.19379, Culture=neutral, PublicKeyToken=4b01fa6e34db77ab, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\Google.Apis.1.9.3\lib\netcore45\Google.Apis.dll</HintPath>
+    <Reference Include="GalaSoft.MvvmLight.Platform">
+      <HintPath>..\..\packages\MvvmLightLibs.5.2.0.0\lib\windows81\GalaSoft.MvvmLight.Platform.dll</HintPath>
     </Reference>
-    <Reference Include="Google.Apis.Auth, Version=1.9.3.19379, Culture=neutral, PublicKeyToken=4b01fa6e34db77ab, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\Google.Apis.Auth.1.9.3\lib\netcore45\Google.Apis.Auth.dll</HintPath>
+    <Reference Include="Google.Apis">

```

Comment:
confirmed using 'sn -T' and 'sn -v' that Google.Apis.dll and Google.Apis.Blogger.v3.dll (e.g.) have a publickeytoken and a signature that verifies.

It looks like the NuGet command line and the NuGet VS extension use different code for adding references. I used the command line, which doesn't appear to add them:
https://github.com/NuGet/NuGet.Client/blob/7f88f41cab8c96dfa289cb5e0c76bfeef6efc93d/src/NuGet.Clients/NuGet.CommandLine/Common/MSBuildProjectSystem.cs#L130

Meanwhile, the VS extension uses Visual Studio COM automation, which probably goes to more trouble:
https://github.com/NuGet/NuGet.Client/blob/d829c8bdce310b560e5e6fd88bd2cece45d06760/src/NuGet.Clients/PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs#L316

My guess is that running in VS would add it again. For now, I'll leave it alone.